### DISCUSSION TITLE
Bots should not attack cuffed pirates.

### DISCUSSION THREAD

**Discussion by ghost** [FakeFishGames/Barotrauma#7295] [Ideas and suggestions]
Just before writing this, I handcuffed a pirate because I expected my crew to leave them alone, and let me get the cash from the "by the book" talent. My security guard bot instead decapitated the cuffed pirate right as I was pulling them to the brig.

Why do they do this? Shouldn't they treat pirates and such similarly to prisoners and ignore them when they're cuffed?

**Comment by itchyOwl** [comment]
Addressed in Regalis11/Barotrauma-development@bb282fd. The bots now treat handcuffed (human) targets as neutralized, unless they are spawned aggressive through an event. I didn't test it on pirates. Please post an issue ticket, if it doesn't work as expected.

**Comment by NaturalBornCamper** [comment]
Still not fixed. Happened to me right now. And every time I heal the prisoners back to life, bots shoot him right away. The only way to prevent is to cancel the order to fight intruders, which is annoying
